app/controllers/file.rb

```diff
@@ -45,7 +45,7 @@ def slim(template, options={})
 				end
 
 				get "/?*", query: "source" do
-					source_view
+					authorize resource, :source?
```

Comment:
When I went to test this, nothing is being rendered? Shouldn't `source_view` come after `authorize resource, :source?`